en/Appendix_A-Glossary.md

```diff
@@ -1,4 +1,5 @@
 # Glossary
+- **Credentials** - Authentication material such as username and password or public and private keys.  
```

Comment:
Revised definition

`Authentication material such as username and password, public and private keys, API keys, or certificates.`

Reply:
Good catch! Will take up the change.